cpu/native/native_cpu.c

```diff
@@ -262,7 +267,8 @@ void native_cpu_init(void)
         err(EXIT_FAILURE, "native_cpu_init: getcontext");
     }
 
-    end_context.uc_stack.ss_sp = __end_stack;
+    end_context.uc_stack.ss_sp = malloc(SIGSTKSZ);
```

Comment:
Why no longer statically allocating? Because

```C
# define SIGSTKSZ sysconf (_SC_SIGSTKSZ)
```

which is not a constant number, but function call issued at runtime. There still is a legacy compat mode in glibc that just defines this as a large constant, but moving the `_GNU_SOURCE` up disabled this compat.